src/main/AuthenticationResponse.ts

```diff
@@ -141,24 +117,12 @@ export default class AuthenticationResponse {
     } else if (!verifyOpts.verification.checkLinkedDomain) {
```

Comment:
this method is missing some parts according to https://openid.net/specs/openid-connect-self-issued-v2-1_0-ID1.html#section-12.1
we can create ticket about that:
1. usage of `i_am_siop` claim in the ID Token
2. The RP MUST validate that the aud (audience) Claim contains the value of the client_id that the RP sent in the Authentication Request as an audience. When the request has been signed, the value might be an HTTPS URL, or a Decentralized Identifier.
3. Verifying that the algorithm of the signature of ID Token is listed in `id_token_signing_alg_values_supported` 
4. validating the nonce